public/client.js

```diff
@@ -2,9 +2,26 @@ $(function () {
 
 	'use strict';
 
+    /*
+     * QUESTION:  I wanted to add underscore here in order to use
+     * _.sortBy to display trucks in alpha order, but I got the 
+     * RefError 'require is not defined'.  What am I doing wrong?
+     *
+     * var _ = require('underscore');    
```

Comment:
Require is a Node function, but this is a client-side script, so you're not using Node here.